filereader/texttemplate/texttemplate.go

```diff
@@ -42,6 +43,25 @@ var funcs2 = template.FuncMap{
 		reg := regexp.MustCompile("[^a-z0-9A-Z_.-]")
 		return reg.ReplaceAllString(data, "_")
 	},
+	"EbsOptimized": func(instanceType string) bool {
+		// Amazon instance series that do not support EBS optimize
+		nonSupportedSeries := map[string]bool{"t1": true, "t2": true}
+		// Amazon instance types that do not support EBS optimize
+		nonSupportedTypes := map[string]bool{
+			"c1.medium": true,
+			"c3.large":  true, "c3.8xlarge": true,
```

Comment:
Interesting way to organize instance types! Good to know gofmt doesn't detect this 😄

Reply:
Yes I was wondering the same after deciding on the formatting - sure enough `make format` didn't complain about it. I think it looked easier on my eyes at the time.